extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts

```diff
@@ -390,15 +390,24 @@ export class MountHdfsDialog extends HdfsDialogBase<MountHdfsProperties> {
 				{
 					component: this.pathInputBox,
 					title: hdfsPathTitle,
-					required: true
+					required: true,
+					layout: {
+						info: localize('mount.hdfsPath.info', "Path to a new (non-existing) directory which you want to associate with the mount")
+					}
 				}, {
 					component: this.remoteUriInputBox,
-					title: localize('mount.remoteUri', "Remote URI"),
-					required: true
+					title: localize('mount.remoteUri.title', "Remote URI"),
+					required: true,
+					layout: {
+						info: localize('mount.remoteUri.info', "The URI to the remote data source. Example for ADLS: abfs://fs@saccount.dfs.core.windows.net/")
+					}
 				}, {
 					component: this.credentialsInputBox,
-					title: localize('mount.credentials', "Credentials"),
-					required: false
+					title: localize('mount.credentials.title', "Credentials"),
+					required: false,
+					layout: {
+						info: localize('mount.credentials.info', "Mount credentials for authentication to remote data source, for access to read the remote data")
```

Comment:
"for access to read" seems a bit wordy to me. trying to figure out why.

Reply:
I'll ask Nellie to review these 😄

Reply:
Yes, can be shortened to: Mount credentials for authentication to remote data source for reads.

Reply:
Done